docs/user/autoyast/reference.md

```diff
@@ -268,9 +268,16 @@ This section is supported.
 | `reg_server_cert`                  | planned   |                             |                                                        |
 | `reg_server_cert_fingerprint`      | planned   |                             |                                                        |
 | `reg_server_cert_fingerprint_type` | planned   |                             |                                                        |
-| `addons`                           | planned   |                             |                                                        |
+| `addons`                           | yes       | `product.addons[]`          | The addon version value is optional, see details below.|
 | `slp_discovery`                    | planned   |                             |                                                        |
 
+If the addon version is not specified in the Agama installation profile it is
+automatically found in the list of the available addons on the server.
```

Comment:
I would write both sentence in the same paragraph. Is just a single piece of text, no need to split.

If you split the second sentence to a second paragraph it should not start with "It is mandatory" because a new paragraph is a new context so "It" is undefined. :-) It should be "The version is mandatory"